baseband/guppi/frame.py

```diff
@@ -0,0 +1,201 @@
+# Licensed under the GPLv3 - see LICENSE
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
+
+import numpy as np
+from astropy.extern import six
+
+from ..vlbi_base.frame import VLBIFrameBase
+from .header import GUPPIHeader
+from .payload import GUPPIPayload
+
+
+__all__ = ['GUPPIFrame']
+
+
+class GUPPIFrame(VLBIFrameBase):
```

Comment:
I think it again may make more sense to base this on `DADAFrame`.

Reply:
Done.